.clang-format

```diff
@@ -0,0 +1,455 @@
+---
+# Language (LanguageKind) clang-format 3.5
+Language:        Cpp
+
+# Based on clang-10 (latest version supported on Ubuntu 20)
+# No BasedOnStyle
+
+# AccessModifierOffset (Integer) clang-format 3.3
+AccessModifierOffset: -2
+
+# AlignAfterOpenBracket (BracketAlignmentStyle) clang-format 3.8
+AlignAfterOpenBracket: DontAlign
+
+# AlignArrayOfStructures (ArrayInitializerAlignmentStyle) clang-format 13
+
+# AlignConsecutiveAssignments (Boolean) clang-format 3.8
+AlignConsecutiveAssignments: false
```

Comment:
I checked and while I agree it is useful in certain circumstances, it makes the diff much bigger. I propose for users to turn clang-format off in case they believe an alignment will improve readability